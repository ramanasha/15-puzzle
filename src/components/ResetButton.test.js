import renderer from 'react-test-renderer';
import React from 'react';
import ResetButton from './ResetButton';

describe('ResetButton', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(<ResetButton className="xyz" onResetClick={jest.fn()} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
