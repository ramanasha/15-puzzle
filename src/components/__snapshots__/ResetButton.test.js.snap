exports[`ResetButton should render correctly 1`] = `
<div
  className="xyz jNFAtU">
  <button
    onClick={[Function]}>
    Reset Game
  </button>
</div>
`;
